packages/notification-center/package.json

```diff
@@ -40,6 +40,10 @@
     "@storybook/react": "^6.4.19",
     "@storybook/testing-library": "0.1.0",
     "@testing-library/react-hooks": "^8.0.1",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/dom": "^9.3.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
RTL for the `notification-center` package... I've added a couple of the unit/integration tests